### PR TITLE
Added library mimesis and changed registration files

### PR DESCRIPTION
**Pull request by alexbeatnik**
In this commit implemented issue #25
To run registration test case added randomize of data
with help of module mimesis. Module mimesis randomize next
data in registration test cases:
- first_name
- last_name
- phone
- email
- password

### CODE REVIEW

File: oct/tests/api/login.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
+from faker import Faker
 from oct.tests import run_testcase
 from oct.tests.api.registration_pattern import UserRegistration, Identity, Credentials
-from oct.tests.web.creating_emails import EmailsGeneration
 
 
 class Login(Testcase):
-    email = EmailsGeneration().creating_full_email()
+    email = Faker().email()
     password = "12345638"
```
Comment (AlekseyDidilov) [inline oct/tests/api/login.py]: @alexbeatnik why you didn't randomize "password"?

Comment (bambula4000) [reply oct/tests/api/login.py]: @AlekseyDidilov , I think it's no sense

File: oct/tests/api/login.py [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 from pyats.topology import Device
+from faker import Faker
 from oct.tests import run_testcase
 from oct.tests.api.registration_pattern import UserRegistration, Identity, Credentials
-from oct.tests.web.creating_emails import EmailsGeneration
 
 
 class Login(Testcase):
-    email = EmailsGeneration().creating_full_email()
+    email = Faker().email()
```
Comment (bambula4000) [inline oct/tests/api/login.py]: It looks much better now, than before.

File: oct/tests/api/login.py (hunk 1 above)
Comment (AlekseyDidilov) [reply oct/tests/api/login.py]: @bambula4000 I think it has sense, this password doesn't cover letters and special characters, which password field accepts.

Comment (AlekseyDidilov) [reply oct/tests/api/login.py]: @alexbeatnik Also, you didn't change default data on 16,17 lines

Comment (alexbeatnik) [reply oct/tests/api/login.py]: @AlekseyDidilov oct/tests/api/login.py  this test case test login not registrstion to test registration i use full randomize to test login i randomize only email so that it can be run several times, it does not make sense to test the registration in the login case test

Comment (AlekseyDidilov) [reply oct/tests/api/login.py]: @alexbeatnik but if you've already imported lib for randomization, why don't use it here also.

Comment (alexbeatnik) [reply oct/tests/api/login.py]: @AlekseyDidilov You can find answer in my last comment

File: .pylintrc [hunk 3]
```diff
@@ -55,3 +55,6 @@ min-public-methods=1
 
 # Maximum number of public methods for a class (see R0904).
 max-public-methods=7
+
+[TYPECHECK]
+ignored-classes=Fysom,Generator
```
Comment (extsoft) [inline .pylintrc]: @alexbeatnik Why did you add this configuration?

Comment (alexbeatnik) [reply .pylintrc]: I had an error that my variable "generator' has no methods like "email", "first_name" and other so i disided to disable this check

File: oct/tests/api/login.py (hunk 1 above)
Comment (alexbeatnik) [reply oct/tests/api/login.py]: @AlekseyDidilov I randomized all data in login testcase

File: .pylintrc (hunk 3 above)
Comment (alexbeatnik) [reply .pylintrc]: @extsoft Fixed, i used another library
